state/l1infotree.go

```diff
@@ -19,53 +20,40 @@ type L1InfoTreeLeaf struct {
 type L1InfoTreeExitRootStorageEntry struct {
 	L1InfoTreeLeaf
 	L1InfoTreeRoot  common.Hash
-	L1InfoTreeIndex uint64
+	L1InfoTreeIndex uint
 }
```

Comment:
L1InfoTreeIndex is `uint32` there are a type defined `L1InfoTreeIndexType` for that

Reply:
I now, It was completely unusefull and it was deleted